### PR TITLE
Added zabbix java gateway docker image

### PR DESCRIPTION
**Pull request by vshlapakov**
Added Docker image for Zabbix Java Gateway (to gather JMX metrics).

### CODE REVIEW

File: zabbix-java-gateway/run.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+logfile=/var/log/zabbix/zabbix_java_gateway.log
+logdir=$(dirname $logfile)
+pidfile=/var/run/zabbix/zabbix_java_gateway.pid
+piddir=$(dirname $pidfile)
+mkdir -p $piddir $logdir
+touch $logfile
+chown -R zabbix:zabbix $piddir $logfile
+/etc/init.d/zabbix-java-gateway start
+sleep 1
+tail -f $logfile --pid=$(cat $pidfile)
```
Comment (lins05) [inline zabbix-java-gateway/run.sh]: Java process could be quite slow to start, is 1 second enough?

Comment (vshlapakov) [inline zabbix-java-gateway/run.sh]: @lins05 it worked for me in most cases, but your question is reasonable, what is a better value from your point of view? 3s/5s?

Comment (lins05) [inline zabbix-java-gateway/run.sh]: I think we can relax it to 3 seconds. Or better, use a while loop until the
pid file exists.

On Mon, Mar 16, 2015 at 10:04 PM, Viktor Shlapakov <notifications@github.com

Comment (dangra) [inline zabbix-java-gateway/run.sh]: may be better to use `-F`  (uppercase *f*) that avoids uncertain sleeps. Another option is to touch the logfile before tailing it.

**Comment by vshlapakov** [comment]
@lins05 Thanks for the idea, done!

**Comment by lins05** [comment]
Looks good to me now.

**Pull request merged**